.github/workflows/linters.yml

```
@@ -0,0 +1,32 @@
+name: Static Analysis (Linting)
+
+# This workflow is triggered on pushes to trunk, and any PRs.
+on:
+    push:
+        branches: [trunk]
+    pull_request:
+
+jobs:
+    check:
+        name: Lint everything
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
+
+            - name: Install NodeJS
+              uses: actions/setup-node@eeb10cff27034e7acf239c5d29f62154018672fd # v3.3.0
+              with:
+                  node-version-file: '.nvmrc'
+                  cache: yarn
+
+            - name: Install dependencies & setup configs
+              run: |
+                  yarn setup:tools
+
+            - name: Lint JavaScript and Styles
+              run: |
+                  yarn lint:js
```

```suggestion
                  yarn lint:css
                  yarn lint:js
```

I deleted the CSS task at some point during my debugging